test/unit/alignment/multiple/align_multiple_test.cpp

```diff
@@ -84,3 +98,133 @@ TEST(align_multiple_test, the_first_aminoacid_test)
+    std::vector<seqan3::dna4_vector> input{"ACGGTGG"_dna4, "ACCGTGCC"_dna4, "GCCGGTGCC"_dna4};
+    seqan3::gap g{};
+    std::vector<std::vector<seqan3::gapped<seqan3::dna4>>> output
+    {
+        {'A'_dna4,        g, 'C'_dna4, 'G'_dna4, 'G'_dna4, 'T'_dna4, 'G'_dna4, 'G'_dna4,        g},
+        {'A'_dna4,        g, 'C'_dna4, 'C'_dna4, 'G'_dna4, 'T'_dna4, 'G'_dna4, 'C'_dna4, 'C'_dna4},
+        {'G'_dna4, 'C'_dna4, 'C'_dna4, 'G'_dna4, 'G'_dna4, 'T'_dna4, 'G'_dna4, 'C'_dna4, 'C'_dna4}
+    };
+
+    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-2},
+                                                                                    seqan3::gap_open_score{-8}}};
+
+    auto result = seqan3::align_multiple(input, cfg);
+
+    EXPECT_RANGE_EQ(output, result);
+}
+
+TEST(align_multiple_test, the_second_gap_score_test)
+{
+    //alignment generated with seqan_tcoffee app: ./seqan_tcoffee -s data/dna2.fa -a dna -g -2 -e -2 -o data/out_g2.fa
+    std::vector<seqan3::dna5_vector> input{"NNTGTNN"_dna5, "GGTNTNNGT"_dna5, "NGTNTGGG"_dna5};
+    seqan3::gap g{};
+    std::vector<std::vector<seqan3::gapped<seqan3::dna5>>> output
+    {
+        {'N'_dna5, 'N'_dna5, 'T'_dna5, 'G'_dna5, 'T'_dna5, 'N'_dna5,        g, 'N'_dna5,        g,        g,        g,        g,        g},
+        {'G'_dna5,        g,        g, 'G'_dna5, 'T'_dna5, 'N'_dna5, 'T'_dna5, 'N'_dna5, 'N'_dna5, 'G'_dna5, 'T'_dna5,        g,        g},
+        {       g, 'N'_dna5,        g, 'G'_dna5, 'T'_dna5, 'N'_dna5, 'T'_dna5,        g,        g, 'G'_dna5,        g, 'G'_dna5, 'G'_dna5}
+    };
+
+    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-2},
+                                                                                    seqan3::gap_open_score{0}}};
+
+    auto result = seqan3::align_multiple(input, cfg);
+
+    EXPECT_RANGE_EQ(output, result);
+}
+
+TEST(align_multiple_test, the_third_gap_score_test)
+{
+    //alignment generated with seqan_tcoffee app: ./seqan_tcoffee -s data/dna3.fa -a dna -g -16 -e -4 -o data/out_g3.fa
+    std::vector<seqan3::dna15_vector> input{"GGGTGGYTG"_dna15, "KTGTGGYTYTG"_dna15, "KTGTYYYTG"_dna15};
+    seqan3::gap g{};
+    std::vector<std::vector<seqan3::gapped<seqan3::dna15>>> output
+    {
+        {'G'_dna15, 'G'_dna15, 'G'_dna15, 'T'_dna15, 'G'_dna15, 'G'_dna15,         g,         g, 'Y'_dna15, 'T'_dna15, 'G'_dna15},
+        {'K'_dna15, 'T'_dna15, 'G'_dna15, 'T'_dna15, 'G'_dna15, 'G'_dna15, 'Y'_dna15, 'T'_dna15, 'Y'_dna15, 'T'_dna15, 'G'_dna15},
+        {'K'_dna15, 'T'_dna15, 'G'_dna15, 'T'_dna15, 'Y'_dna15, 'Y'_dna15,         g,         g, 'Y'_dna15, 'T'_dna15, 'G'_dna15}
+    };
+
+    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-4},
+                                                                                    seqan3::gap_open_score{-14}}};
```

Comment:
To have -16 as gap open score in seqan2, we need to put -12 here, no?
```suggestion
                                                                                    seqan3::gap_open_score{-12}}};
```

Reply:
definitively yes ! 👍

Reply:
I thought about finding an alignment where it actually makes a difference if the gap_open_score varies. But this is easier when go and g scores are closer to each other and this is already covered in at least one of the other tests. So I think this should be fine :)